web_responsive/static/src/js/web_responsive.js

```diff
@@ -112,6 +113,14 @@ odoo.define('web_responsive', function (require) {
             return this._super.apply(this, arguments);
         },
 
+        /**
+         * @override
+         */
+        _onAppsMenuItemClicked: function (ev) {
+            this._super.apply(this, arguments);
+            ev.preventDefault();
```

Comment:
Add a comment explaining why you do this.

Reply:
https://github.com/OCA/web/commit/ce0391a0a0a21de4cc3caffa6829a99437e3db0e